tests/popup-test.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * Copyright 2017 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+require("../readiness-tool/popup"); 
+
+
```

Comment:
A couple ideas for unit tests (add TODOs here to track them):

- Test that apps.json is a valid JSON format.
- Test `findDetectedApps()` with test HTML markup for the top 5 analytics/ads vendors.
- Test `isSupported()` for the top 10 analytics/ads vendors.